odoo/addons/test_pylint/tests/test_pylint.py

```diff
@@ -27,6 +27,14 @@ class TestPyLint(TransactionCase):
         'W0101',  # unreachable code
 
         'mixed-indentation',
+        'deprecated-module',
+    ]
+
+    BAD_MODULES = [
+        'pdb',
+        'pudb',
+        'ipdb',
+        'wdb',
```

Comment:
@KangOl done here, as suggested by nim

Reply:
Maybe we could source that modules list straight from the SUPPORTED_DEBUGGER set, that way if we update one the other will be in sync?

Reply:
Good idea but isn't it the root of all evil ?
Anyway, I do it